comp/core/tagger/filters/filter_builder.go

```diff
@@ -0,0 +1,74 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Were this file and `filters.go` meant to be included in this PR? The title and the PR description don't seem to imply it.